examples/tagger_train.go

```diff
@@ -1,3 +1,5 @@
+// +build ignore
```

Comment:
This and a set of other files seem to be not related to String.  This is not the problem, but let's try to keep things atomic.